backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -664,91 +233,6 @@ func TestCreateRun_InvalidWorkflowSpec(t *testing.T) {
 	assert.Contains(t, err.Error(), "Failed to unmarshal workflow spec manifest")
 }
 
-func TestCreateRun_NullWorkflowSpec(t *testing.T) {
```

Comment:
add a comment why it was removed to help with future KFP merges? the one above might not show in the Git conflict resolution